savant_core/src/telemetry.rs

```diff
@@ -1,29 +1,37 @@
 use opentelemetry::global;
+use opentelemetry_jaeger_propagator::Propagator;
+use opentelemetry_otlp::WithExportConfig;
 use opentelemetry_sdk::propagation::TraceContextPropagator;
 use opentelemetry_sdk::trace::{Config, TracerProvider};
 use opentelemetry_sdk::Resource;
+use opentelemetry_semantic_conventions::resource::{SERVICE_NAME, SERVICE_NAMESPACE};
+use opentelemetry_stdout::SpanExporter;
 
 pub fn init_jaeger_tracer(service_name: &str, endpoint: &str) {
```

Comment:
Please rename and update everywhere to let users know that this is a generic tracer, not jaeger: `init_otlp_tracer`.

Also, check it really works and update the test: https://github.com/insight-platform/savant-rs/blob/main/python/pipeline.py

Reply:
Please use batch exporter for efficiency reasons.

Reply:
I have checked only with Jaeger and decided to leave context propagation in Jaeger format for backward compatibility.

Do we need to make context propagation format configurable?

 If we want to make this fully configurable, then the configuration will be rather complex starting that both grpc and http should be supported.

About test: do you want to have an integration test that will deploy OLTP backend and check that it works?

Reply:
I mostly care about auth and compression.
Propagation format: yes, optional
Test: yes